CMakeLists.txt

```diff
@@ -22,12 +22,19 @@ if (APPLE)
     SET(Qt5Charts_DIR "/usr/local/opt/qt/lib/cmake/Qt5Charts")
     find_package(Qt5Core REQUIRED)
     find_package(Qt5Gui REQUIRED)
-    set(GTEST_LIB /usr/local/lib/libgtest.a /usr/local/lib/libgtest_main.a)
-else (NOT APPLE)
+    set(GTEST_LIB
+            /usr/local/lib/libgtest.a
+            /usr/local/lib/libgtest_main.a
+            /usr/local/lib/libgmock.a
+            /usr/local/lib/libgmock_main.a
+            )
+else(NOT APPLE)
     SET(Qt5Widgets_DIR "/usr/include/x86_64-linux-gnu/qt5/QtWidgets")
     SET(Qt5Charts_DIR "/usr/include/x86_64-linux-gnu/qt5/QtCharts")
-    set(GTEST_LIB PRIVATE gtest)
-endif ()
+    set(GTEST_LIB
+            PRIVATE gtest
+            PRIVATE gmock)
```

Comment:
good catch. I have disabled it in a test for now but Google Mock is something really useful for writing tests. It allows you to temporarily override functions. In that case you can test functions that depend on that functions. 

So in my case I could create a scorematrix with artificial scores, testing whether the mapping was correct, instead of generating a worldstate and depending on a lot of functions from world and robot.

I disabled it because I needed some more flexibility and did it manually (see the test). but google mock is very nice to have.